Vagrantfile

```diff
@@ -44,6 +44,7 @@ Vagrant.configure(2) do |config|
       run: "always"
 
     distgit.vm.provision "shell",
+      privileged: false,
       inline: "cd /vagrant/ && tito build --test --rpm",
       run: "always"
```

Comment:
You applied the wrong patch :-(
It doesn't fix the EPEL machines.

Reply:
Isn't it fixed in the next commit?

Reply:
See the patch here https://github.com/release-engineering/dist-git/files/4638660/0001-Build-the-dist-git-package-as-non-root-user.patch.txt , it contains this line as well as changes for the other two machines.

Reply:
Yes, I included those changes in a new commit.

Reply:
https://github.com/release-engineering/dist-git/pull/33/commits/c0ac98713504d7d4f7bbb5c59807debba437664a